### PR TITLE
Wp/celeste

### PR DESCRIPTION
**Pull request by celestepalet**
Hi, add the TC and the libraries for put method. Regards

### CODE REVIEW

File: .env
Comment (Fabian2025) [inline .env:1]: please add a new line at the end of file.

File: .gitignore
Comment (Fabian2025) [inline .gitignore:1]: please add a new line in the end of file.

**Comment by Fabian2025** [comment]
please add new lines in the .env and .gitignore files

**Comment by celestepalet** [comment]
Hi Fabian, I added the requested lines.

File: .gitignore
Comment (celestepalet) [reply .gitignore:1]: ready

File: .env
Comment (celestepalet) [reply .env:1]: ready

**Review by Fabian2025 — APPROVED** [review]

**Pull request merged**